### PR TITLE
composer: add Polish translation, update page

### PR DESCRIPTION
**Pull request by TheBulaDev**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/composer.md [hunk 1]
```diff
@@ -7,23 +7,23 @@
...
 
 `composer install`
 
-- Uninstall a package from this project, removing it as a dependency from `composer.json`:
+- Uninstall a package from this project, removing it as a dependency from `composer.json` and `composer.lock` files:
 
 `composer remove {{user/package}}`
 
-- Update all the dependencies in this project's `composer.json` and note versions in `composer.lock` file:
+- Update all the dependencies in this project's `composer.json` and note a new versions in `composer.lock` file:
```
Comment (kbdharun) [inline pages/common/composer.md:22]:
```suggestion
- Update all the dependencies in this project's `composer.json` and note new versions in the `composer.lock` file:
```

File: pages/common/composer.md [hunk 2]
```diff
@@ -7,23 +7,23 @@
...
 
 `composer update`
 
-- Update composer lock only after updating `composer.json` manually:
+- Update `composer.lock` file only after updating `composer.json` manually:
```
Comment (kbdharun) [inline pages/common/composer.md:26]:
```suggestion
- Update the `composer.lock` file only after updating `composer.json` manually:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/common/composer.md (hunk 2 above)
Comment (TheBulaDev) [reply pages/common/composer.md]: Thanks for your suggestions

**Review by kbdharun — APPROVED** [review]

File: pages/common/composer.md [hunk 3]
```diff
@@ -7,23 +7,23 @@
 
 `composer init`
 
-- Add a package as a dependency for this project, adding it to `composer.json`:
+- Add a package as a dependency for this project, adding it to `composer.json` file:
```
Comment (acuteenvy) [inline pages/common/composer.md:10]:
```suggestion
- Add a package as a dependency for this project, adding an entry to `composer.json`:
```

File: pages/common/composer.md [hunk 4]
```diff
@@ -7,23 +7,23 @@
...
 
 `composer require {{user/package}}`
 
 - Install all the dependencies in this project's `composer.json` and create `composer.lock`:
 
 `composer install`
 
-- Uninstall a package from this project, removing it as a dependency from `composer.json`:
+- Uninstall a package from this project, removing it as a dependency from `composer.json` and `composer.lock` files:
```
Comment (acuteenvy) [inline pages/common/composer.md:18]:
```suggestion
- Uninstall a package from this project, removing it as a dependency from `composer.json` and `composer.lock`:
```

File: pages/common/composer.md [hunk 5]
```diff
@@ -7,23 +7,23 @@
...
 
 `composer remove {{user/package}}`
 
-- Update all the dependencies in this project's `composer.json` and note versions in `composer.lock` file:
+- Update all the dependencies in this project's `composer.json` and note new versions in the `composer.lock` file:
```
Comment (acuteenvy) [inline pages/common/composer.md:22]:
```suggestion
- Update all the dependencies in this project's `composer.json` and note new versions in `composer.lock`:
```

File: pages/common/composer.md [hunk 6]
```diff
@@ -7,23 +7,23 @@
...
 
 `composer update`
 
-- Update composer lock only after updating `composer.json` manually:
+- Update the `composer.lock` file only after updating `composer.json` manually:
```
Comment (acuteenvy) [inline pages/common/composer.md:26]:
```suggestion
- Update only `composer.lock` after updating `composer.json` manually:
```

File: pages.pl/common/composer.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+> Menadżer pakietów dla projektów PHP.
+> Więcej informacji: <https://getcomposer.org/>.
+
+- Interaktywnie utwórz plik `composer.json`:
+
+`composer init`
+
+- Dodaj pakiet do zależności tego projektu, dodając wpis do pliku `composer.json`:
+
+`composer require {{uzytkownik/pakiet}}`
```
Comment (acuteenvy) [inline pages.pl/common/composer.md:12]:
```suggestion
- Dodaj pakiet do zależności tego projektu, dodając wpis do `composer.json`:

`composer require {{użytkownik/pakiet}}`
```

File: pages.pl/common/composer.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Zainstaluj wszystkie zależności z projektowego `composer.json` i utwórz `composer.lock`:
+
+`composer install`
+
+- Odinstaluj pakiet z tego projektu, usuń go jako zaleśność z plików `composer.json` i `composer.lock`:
+
+`composer remove {{uzytkownik/pakiet}}`
```
Comment (acuteenvy) [inline pages.pl/common/composer.md:20]:
```suggestion
- Odinstaluj pakiet z tego projektu, usuń go jako zależność z `composer.json` i `composer.lock`:

`composer remove {{użytkownik/pakiet}}`
```

File: pages.pl/common/composer.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+- Zaktualizuj wszystkie pakiety z projektowego `composer.json` i zanotuj nową wersję w pliku `composer.lock`:
```
Comment (acuteenvy) [inline pages.pl/common/composer.md:22]:
```suggestion
- Zaktualizuj wszystkie pakiety z projektowego `composer.json` i zanotuj nową wersję w `composer.lock`:
```

File: pages.pl/common/composer.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+`composer update`
+
+- Zaktualizuj tylko plik `composer.lock` po ręcznej aktualizacji `composer.json`:
+
+`composer update --lock`
```
Comment (acuteenvy) [inline pages.pl/common/composer.md:28]:
```suggestion
- Zaktualizuj tylko `composer.lock` po ręcznej aktualizacji `composer.json`:

`composer update --lock`
```

File: pages.pl/common/composer.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+
+- Dowiedz się więcej o powodach dlaczego zależność nie może zostać zainstalowana:
+
+`composer why-not {{uzytkownik/pakiet}}`
```
Comment (acuteenvy) [inline pages.pl/common/composer.md:32]:
```suggestion
`composer why-not {{użytkownik/pakiet}}`
```

**Review by TheBulaDev — COMMENTED** [review]
fix cr 2

File: pages.pl/common/composer.md [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+
+- Dodaj pakiet do zależności tego projektu, dodając wpis do `composer.json`:
+
+`composer require {{użytkownik/pakiet}}`
+
+- Zainstaluj wszystkie zależności z projektowego `composer.json` i utwórz `composer.lock`:
+
+`composer install`
+
+- Odinstaluj pakiet z tego projektu, usuń go jako zależność z `composer.json` i `composer.lock`:
```
Comment (acuteenvy) [inline pages.pl/common/composer.md:18]:
```suggestion
- Odinstaluj pakiet z tego projektu, usuwając go jako zależność z `composer.json` i `composer.lock`:
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**